storages/sled-storage/src/key.rs

```diff
@@ -4,6 +4,20 @@ const TEMP_DATA: &str = "temp_data/";
 const TEMP_SCHEMA: &str = "temp_schema/";
 const TEMP_INDEX: &str = "temp_index/";
 
+pub fn data_prefix(table_name: &str) -> String {
+    format!("data/{table_name}/")
+}
+
+pub fn data(table_name: &str, key: Vec<u8>) -> IVec {
+    let key = data_prefix(table_name)
+        .into_bytes()
+        .into_iter()
+        .chain(key.into_iter())
+        .collect::<Vec<_>>();
```

Comment:
https://docs.rs/sled/latest/sled/struct.IVec.html#impl-FromIterator%3Cu8%3E

Can you delete `collect` and use from as an iterator?

Reply:
Oh, thanks a lot. This looks that we can apply this not only for `key::data`, but also all other functions in `key.rs` module.
I'll make this as a separate PR for this.

Reply:
Ok i will check